dev-app/src/App.tsx

```diff
@@ -175,7 +175,9 @@ export default function App() {
       }
     }
     if (Platform.OS === 'android') {
-      handlePermissions();
+      setTimeout(() => {
+        handlePermissions();
+      }, 1000); // delay of 1 second
```

Comment:
not new for this PR so no need to block on it but why do we need this in the app layer? Won't the native SDK request and prompt for the permissions when needed? 🤔

Reply:
The native Android SDK doesn't prompt for permissions itself, it requires the integration to do so.

Reply:
ah, TIL! thanks 👍